packages/@icon-magic/distribute/src/create-icon-template.ts

```diff
@@ -39,7 +39,7 @@ export async function createHbs(
 
         return {
           ElementNode(node) {
-            if (node.tag === 'svg') {
+            if (node.tag === 'svg' && node.attributes.find(attr => attr.name === 'xmlns')) {
```

Comment:
I _thought_ these come from the source Joe gives us and cannot be relied upon. Maybe better to check that there are no children `svg` tags?

Reply:
I remove it in `svg-light-dark` since we don't actually need it for anything.

We can't check for no children `svg` tags because:

for mixed variants the parent svg _will_ have children svgs
for regular light variants there won't be children svgs (since no dark variant)